src/Autosuggest.js

```diff
@@ -235,15 +252,19 @@ class Autosuggest extends Component {
           case 'Enter': {
             const focusedSuggestion = this.getFocusedSuggestion();
 
-            if (focusedSuggestion !== null) {
+            if (focusedSuggestion) {
```

Comment:
Update: After incorporating the changes [here](https://github.com/moroshko/react-autosuggest/commit/c2cc05484a0b8a459f65f115177aa8be5dec7d90) we can explicitly check for null since `maybeSelectFirstSuggestion` isn't called for every `onChange` anymore.